neo/public/javascripts/map/controllers.js

```diff
@@ -323,18 +364,33 @@ angular.module('cloudberry.map', ['leaflet-directive', 'cloudberry.common'])
         // draw
         $scope.polygons.statePolygons.setStyle(style);
       } else if ($scope.status.logicLevel == "county" && $scope.geojsonData.county) {
-        angular.forEach($scope.geojsonData.county.features, function(d) {
+          angular.forEach($scope.geojsonData.county.features, function(d) {
+            if (d.properties.count)
```

Comment:
Added a FIXME comment here. I'll try to combine them later as I know more about JS